LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -322,7 +322,10 @@ public virtual async Task<bool> InitializeAsync(NetworkServerConfiguration confi
                     NetID = twin.Properties.Reported[TwinProperty.NetID].Value as string;
 
                 if (twin.Properties.Reported.Contains(TwinProperty.DevNonce))
-                    DevNonce = twin.Properties.Reported[TwinProperty.DevNonce].Value as string;
+                {
+                    var devNonceString = twin.Properties.Reported[TwinProperty.DevNonce].Value as string;
+                    DevNonce = ushort.TryParse(devNonceString, out var d) ? new DevNonce(d) : null;
```

Comment:
I am surprised that analyzers did not pick up (here and elsewhere in this PR) on using the overload that requires `NumberStyles` and `IFormatProvider`, like so:

```c#
DevNonce = ushort.TryParse(devNonceString, NumberStyles.None, CultureInfo.InvariantCulture, out var d) ? new DevNonce(d) : null;
```

Reply:
Addressed this in 1cf5d208c6122f300c2223dc70ecb893aba9928d